packages/wasm-terminal/src/wasm-terminal.ts

```diff
@@ -1,6 +1,7 @@
 // The Wasm Terminal
 import xterm from "xterm";
-const Terminal = xterm.Terminal;
+import * as xtermDefault from "xterm";
```

Comment:
`import *` is what caused the build errors from last time. 🤔 

Also, this would import the default export twice? If you want to pair on this, let me know 😄 Maybe there is a reason this is working in parcel (if it is working)?

Reply:
So basically, this was the line that was breaking the wasm-shell example.
And after experimenting for 30 min I got this hack to get it working (both having a successful `npm run build` and a successful `npm run dev`)